requirements.txt

```diff
@@ -5,3 +5,4 @@ enum34>=1.0.4
 protobuf>=3.5.0.post1
 six>=1.10
 wheel>=0.29
+Sphinx~=1.8.1
```

Comment:
Do we really want to require every user install Sphinx? 99.9999% of users won't want to build documentation. Is there another way to ensure this package is installed in gRPC doc generation environment?

Reply:
How about create another requirement file? Hardcode the version in scripts is not ideal.

Reply:
I understand, but this is just one and will forever be one script that needs Sphinx. Why add an additional level of indirection?